### PR TITLE
Rearrange source tree

### PR DESCRIPTION
**Pull request by hdevalence**
After #83, there's no need for the `aggregated_range_proof` tree, but I made this as a seperate PR so that it's easier to review the substantive changes of #83 vs the superficial changes of this one.

### CODE REVIEW

**Comment by hdevalence** [comment]
@cathieyun I think that this may move things out from under your notes, but I can do the work of fixing that branch relative to this one.

**Comment by cathieyun** [comment]
No worries - it's not a big change, I can fix that up :)

File: src/range_proof/mod.rs [hunk 1]
```diff
@@ -1,35 +1,74 @@
...
-impl SinglePartyAggregator {
-    /// Create an aggregated rangeproof of multiple values.
+impl RangeProof {
+    /// Create a rangeproof for a given pair of value `v` and
+    /// blinding scalar `v_blinding`.
     ///
-    /// This performs the same proof aggregation MPC protocol, but
-    /// with one party playing all roles.
+    /// XXX add doctests
+    pub fn prove_multiple_single<R: Rng>(
```
Comment (oleganza) [inline src/range_proof/mod.rs]: shouldn't it be named `prove_single`?

File: src/range_proof/mod.rs [hunk 2]
```diff
@@ -70,17 +109,165 @@ impl SinglePartyAggregator {
...
+    ///
+    /// XXX add doctests
+    pub fn verify<R: Rng>(
+        &self,
+        value_commitments: &[RistrettoPoint],
+        gens: GeneratorsView,
+        transcript: &mut ProofTranscript,
+        rng: &mut R,
+        n: usize,
+        m: usize,
```
Comment (oleganza) [inline src/range_proof/mod.rs]: maybe use `m = value_commitments.len()` instead of an explicit parameter?

**Pull request merged**